orchestra/management/commands/migrate_certifications.py

```diff
@@ -0,0 +1,26 @@
+from django.core.management.base import BaseCommand
+from orchestra.workflow.certifications import migrate_certifications
+
+
+class Command(BaseCommand):
+    help = 'Migrates specified certifications across workflows.'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'workflow_src',
+            help='Slug for the source workflow from which to migrate '
+                 'certifications.')
+        parser.add_argument(
+            'workflow_dst',
+            help='Slug for the destination workflow to which certifications '
+                 'will be migrated.')
+        parser.add_argument(
+            '--certifications', nargs='+',
+            help=('Certification slugs to migrate. If not specified, '
+                  'migration of all certifications will be attempted.'))
+
+    def handle(self, *args, **options):
+        migrate_certifications(
+            options['workflow_src'],
+            options['workflow_dst'],
+            options['certifications'])
```

Comment:
maybe print some messages so the user knows it's working? (the `loadworkflow` command is silent, and that kind of annoys me).

Print from a management command with `print(msg, file=self.stdout)` (or `self.stderr`, if applicable).